### ISSUE TITLE
anaconda installation

### ISSUE THREAD

**Issue by nevadarain** [ContinuumIO/anaconda-issues#3947]
Can anaconda be installed on windows 10 with a home directory that has spaces?

**Comment by mingwandroid** [comment]
Yes but it is strongly discouraged. Many things that you can install with conda will fail in that situation.

**Comment by nevadarain** [comment]
so @mingwandroid how can  I install it without it failing, since i cant change the home directory name?

**Comment by nilavra** [comment]
I was having similar problems by installing Miniconda in my Users folder (which contains a space). So I did a complete uninstall and reinstalled Miniconda in C:\Miniconda3 (as is suggested [here](https://support.anaconda.com/customer/en/portal/articles/2913013-%22error-cannot-install-into-directories-with-spaces%22)). 
This new installation should not have anything to do with my User directory, as it is (should be?) completely detached. Yet somehow whenever I try to activate an environment, I get similar errors like [this](https://github.com/ContinuumIO/anaconda-issues/issues/1029#issuecomment-489586375).

Here's my output:

```
D:\ALL_WORKSPACES\vizwiz-api> conda activate nb
Access is denied.
The system cannot find the file C:\Users\Nilavra.
Could Not Find C:\Users\Nilavra Bhattacharya\AppData\Local\Temp\conda-9484.tmp

D:\ALL_WORKSPACES\vizwiz-api> conda --version
conda 4.6.14

D:\ALL_WORKSPACES\vizwiz-api> where conda
C:\Miniconda3\condabin\conda.bat
C:\Miniconda3\Scripts\conda.exe
```
